lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2795,27 +2771,47 @@ instance Buildable a => Show (ShowBuildable a) where
 instance Arbitrary (Hash "Datum") where
     arbitrary = pure $ Hash $ BS.pack $ replicate 28 0
 
+instance Arbitrary (Cardano.AddressInEra Cardano.AlonzoEra) where
+    arbitrary = genAddressInEra Cardano.AlonzoEra
+
+instance Arbitrary (Cardano.TxOutValue Cardano.AlonzoEra) where
+    arbitrary = Cardano.TxOutValue Cardano.MultiAssetInAlonzoEra
+        <$> genValueForTxOut
+    shrink _ = [] -- TODO
+
+instance Arbitrary (Cardano.TxOutDatum ctx Cardano.AlonzoEra) where
+    arbitrary = genTxOutDatum Cardano.AlonzoEra
+
+instance Arbitrary (Cardano.TxOut ctx Cardano.AlonzoEra) where
+    arbitrary = genTxOut Cardano.AlonzoEra
+    shrink (Cardano.TxOut addr val dat) = tail
+        [ Cardano.TxOut addr' val' dat'
+        | addr' <- prependOriginal shrink addr
+        , val' <- prependOriginal shrink val
+        , dat' <- prependOriginal shrink dat
+        ]
```

Comment:
@Anviking 

Just curious, have we tested this shrinker anywhere?

The following also seems a possibility:
```suggestion
    shrink = shrinkMapBy
        (Compatibility.toCardanoTxOut ShelleyBasedEraAlonzo)
        (Compatibility.fromCardanoTxOut)
        (shrink)
```

Reply:
> Just curious, have we tested this shrinker anywhere?

I tried it yesterday, and it seemed to work in that case.